pkg/util/github/release.go

```diff
@@ -0,0 +1,23 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/devstream-io/devstream/pkg/util/log"
+)
+
+func (c *Client) LatestReleaseTagName() (string, error) {
```

Comment:
this func is a little incomprehensible  by name, you can add a comment to describe this func

Reply:
How about `GetLatestReleaseTagName`?

Reply:
that's great and more clear